website/docs/troubleshooting/troubleshooting-podman-on-linux.md

```diff
@@ -3,3 +3,7 @@ sidebar_position: 40
 ---
 
 # Troubleshooting Podman on Linux
+
+## Podman Desktop does not manage native Podman
+
+Podman Desktop manages Podman machines in virtual machines, and does not handle native Podman installation configuration files.
```

Comment:
```suggestion
On Linux, Podman runs natively on the host, rather than in a virtual machine.	
Podman Desktop manages Podman machines in virtual machines, and does not handle native Podman installation configuration files.
```